app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+class User < ApplicationRecord
+  before_save{self.email = email.downcase}
+  validates :name, presence: true, length: {maximum: 50}
```

Comment:
1. thêm line trống sau line 2
2. `self.email = email.downcase`
thành `email.downcase!`
3. các giá trị `50` `6` `255` 
em  khai báo vào file settings nhé
để làm dc em tìm hiểu `gem config in rails` nhé